### ISSUE TITLE
ember g model testM1 generate only coffee file. No javascript file getting generated

### ISSUE THREAD

**Issue by MadMaxEmber** [ember-cli/ember-cli#8745]
Thank you for taking the time to open an issue!

For bug reports please include the following section in your issue details.
If you include instructions on how to reproduce the bug or a failing test case
it will make it easier for us to track down the issue you're having.

---

Output from `ember version --verbose && npm --version && yarn --version`:
```
[Replace this line with the output.]
```

**Comment by dtolios** [comment]
I'm also not able to generate models using ember-cli. I'm on Windows 10, using git bash, ember version 3.11

My output looks like this:
```
$ ember generate model person
installing model
installing model-test
```
No js files are generated.

**Comment by pichfl** [comment]
Can you post the content of your projects `package.json` here? (feel free to edit out the package name if necessary)

**Comment by dtolios** [comment]
Sure thing:
```
{
  "name": "league-client",
  "version": "1.0.0",
  "private": true,
  "description": "Small description for league-client goes here",
  "repository": "",
  "license": "MIT",
  "author": "Dimitri Tolios <dtolios09@gmail.com>",
  "directories": {
    "doc": "doc",
    "test": "tests"
  },
  "scripts": {
    "build": "ember build",
    "lint:hbs": "ember-template-lint .",
    "lint:js": "eslint .",
    "start": "ember serve",
    "test": "ember test"
  },
  "devDependencies": {
    "@ember/jquery": "^0.6.0",
    "@ember/optional-features": "^0.7.0",
    "broccoli-asset-rev": "^3.0.0",
    "ember-ajax": "^5.0.0",
    "ember-cli": "~3.11.0",
    "ember-cli-app-version": "^3.2.0",
    "ember-cli-babel": "^7.7.3",
    "ember-cli-dependency-checker": "^3.1.0",
    "ember-cli-eslint": "^5.1.0",
    "ember-cli-htmlbars": "^3.0.1",
    "ember-cli-htmlbars-inline-precompile": "^2.1.0",
    "ember-cli-inject-live-reload": "^1.8.2",
    "ember-cli-mirage": "^1.1.0",
    "ember-cli-sass": "^10.0.1",
    "ember-cli-sri": "^2.1.1",
    "ember-cli-template-lint": "^1.0.0-beta.1",
    "ember-cli-uglify": "^2.1.0",
    "ember-data": "~3.11.0",
    "ember-export-application-global": "^2.0.0",
    "ember-load-initializers": "^2.0.0",
    "ember-maybe-import-regenerator": "^0.1.6",
    "ember-qunit": "^4.4.1",
    "ember-resolver": "^5.0.1",
    "ember-source": "~3.11.1",
    "ember-welcome-page": "^4.0.0",
    "eslint-config-prettier": "^6.0.0",
    "eslint-plugin-ember": "^6.2.0",
    "eslint-plugin-node": "^9.0.1",
    "eslint-plugin-prettier": "^3.1.0",
    "loader.js": "^4.7.0",
    "prettier": "^1.18.2",
    "qunit-dom": "^0.8.4",
    "sass": "^1.22.7"
  },
  "engines": {
    "node": "8.* || >= 10.*"
  },
  "main": "app.js"
}
{
  "name": "",
  "version": "1.0.0",
  "private": true,
  "repository": "",
  "license": "MIT",
  "author": "Dimitri Tolios <dtolios09@gmail.com>",
  "directories": {
    "doc": "doc",
    "test": "tests"
  },
  "scripts": {
    "build": "ember build",
    "lint:hbs": "ember-template-lint .",
    "lint:js": "eslint .",
    "start": "ember serve",
    "test": "ember test"
  },
  "devDependencies": {
    "@ember/jquery": "^0.6.0",
    "@ember/optional-features": "^0.7.0",
    "broccoli-asset-rev": "^3.0.0",
    "ember-ajax": "^5.0.0",
    "ember-cli": "~3.11.0",
    "ember-cli-app-version": "^3.2.0",
    "ember-cli-babel": "^7.7.3",
    "ember-cli-dependency-checker": "^3.1.0",
    "ember-cli-eslint": "^5.1.0",
    "ember-cli-htmlbars": "^3.0.1",
    "ember-cli-htmlbars-inline-precompile": "^2.1.0",
    "ember-cli-inject-live-reload": "^1.8.2",
    "ember-cli-mirage": "^1.1.0",
    "ember-cli-sass": "^10.0.1",
    "ember-cli-sri": "^2.1.1",
    "ember-cli-template-lint": "^1.0.0-beta.1",
    "ember-cli-uglify": "^2.1.0",
    "ember-data": "~3.11.0",
    "ember-export-application-global": "^2.0.0",
    "ember-load-initializers": "^2.0.0",
    "ember-maybe-import-regenerator": "^0.1.6",
    "ember-qunit": "^4.4.1",
    "ember-resolver": "^5.0.1",
    "ember-source": "~3.11.1",
    "ember-welcome-page": "^4.0.0",
    "eslint-config-prettier": "^6.0.0",
    "eslint-plugin-ember": "^6.2.0",
    "eslint-plugin-node": "^9.0.1",
    "eslint-plugin-prettier": "^3.1.0",
    "loader.js": "^4.7.0",
    "prettier": "^1.18.2",
    "qunit-dom": "^0.8.4",
    "sass": "^1.22.7"
  },
  "engines": {
    "node": "8.* || >= 10.*"
  },
  "main": "app.js"
}
```

**Comment by proteamer** [comment]
Hi, I faced with the same issue I can reproduce it for new project:
```
$ ember -v
ember-cli: 3.11.0
node: 10.16.0
os: darwin x64
$ ember new demo-app --yarn
```

**Comment by rwjblue** [comment]
Ember Data 3.11 has a bug here, the blueprints are not working ATM but we are working on fixing.

**Comment by MadMaxEmber** [comment]
I am using ember V1.1
After executing command to create model instead of js file a coffee file is getting created

$ ember g model employee

![image](https://user-images.githubusercontent.com/52817696/61768333-cb92ab80-ae04-11e9-9a1a-906c6e0c565d.png)


Here is the package.json configuration
![image](https://user-images.githubusercontent.com/52817696/61768294-ab62ec80-ae04-11e9-9874-a27240ebb2a8.png)

**Comment by pichfl** [comment]
@MadMaxEmber Your `package.json` includes `ember-cli-coffeescript`, which brings its own set of blueprints. As long as you have that add-on, the behavior you describe is intended.

If you don't want this, the official documentation of the add-on includes a way to turn these blueprints off, see https://github.com/kimroen/ember-cli-coffeescript#blueprints

**Comment by dtolios** [comment]
@rwjblue Is there an open issue or place to track progress other than here?

**Comment by efx** [comment]
The mentioned ember-data blueprint bug was fixed here: https://github.com/emberjs/data/pull/6286 and was released in 3.11.4: https://github.com/emberjs/data/blob/v3.11.4/CHANGELOG.md. So I think we can close this issue. AFAIK, no one reported the blueprint issue in https://github.com/emberjs/data.

**Comment by rwjblue** [comment]
Thank you @efx , closing...
